Makefile

```diff
@@ -125,6 +126,8 @@ dockerpush: dockerbuild-go
 
 	docker push spacemeshos/$(DOCKER_IMAGE_NAME):$(BRANCH)
 	docker push spacemeshos/$(DOCKER_IMAGE_NAME):$(SHA)
+	docker tag $(DOCKER_IMAGE_NAME):$(BRANCH) spacemeshos/$(TESTS_DOCKER_IMAGE_NAME):$(BRANCH)
```

Comment:
rename DOCKER_IMAGE_NAME to DOCKER_REPO or something

Reply:
Done in: https://github.com/spacemeshos/go-spacemesh/pull/773